Assets/Scripts/Game/UserInterfaceWindows/DaggerfallRestWindow.cs

```diff
@@ -312,17 +315,17 @@ bool TickVitals()
 
             playerEntity.TallySkill((short)Skills.Medical, 1);
 
+            return IsPlayerFullyHealed();
+        }
+
+        bool IsPlayerFullyHealed()
+        {
             // Check if player fully healed
             // Will eventually need to tailor check for character
             // For example, sorcerers cannot recover magicka from resting
-            if (playerEntity.CurrentHealth == playerEntity.MaxHealth &&
```

Comment:
This is my preferred style here. Please do not make style changes that are functionally identical. Thank you. :)